src/commands/publish.ts

```diff
@@ -189,12 +201,14 @@ const release = async (note: string): Promise<void> => {
     try {
         const res: AxiosResponse = await releaseBlockRequest(id, note, version);
 
+        logResponse(res);
+
         updateBlockSettingsFile({
             activeVersion: version,
         });
 
         logSuccess(`
-            Released ${displayName} v${version} for production
+            Released ${displayName} v${version}
```

Comment:
Glad you removed this, that part of the message was confusing when publishing to sandbox.